content/docs/3.0/concepts/glossary.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Glossary
+summary: Glossaries about TiKV.
+menu:
+    docs:
+        parent: Concepts
+---
+
+## L
+
+### leader/follower/learner
```

Comment:
Maybe these can be split up

Reply:
You mean they can be introduced separately?

Reply:
I think they can be separate entries in this glossary?